docs/i2c.rst

```diff
@@ -0,0 +1,43 @@
+I²C
+***
+
+.. py:module:: microbit.i2c
+
+The ``i2c`` module lets you communicate with devices connected to your board
+using the I²C bus protocol. There can be multiple slave devices connected at
+the same time, and each one has its own unique address, that is either fixed
+for the device or configured on it. Your board acts as the I²C master.
```

Comment:
This paragraph assumes the reader is familiar with the definition of what an I2C master and slave are. Perhaps it could be rephrase to initialy say you can connect devices to the microbit, and then define that the master would be the "controller" and the slaves are the the "controlled" devices.